README.md

```diff
@@ -1,2 +1,53 @@
-# Portfolio
-My portfolio
+# Project Name
+Portfolio: set up and mobile first
+
+
+## Built With
+HTML
+CSS
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
```

Comment:
- [ ] As requested by the previous code reviewer, Microverse has updated this previous readme, and now we have a [NEW README FILE](https://github.com/microverseinc/readme-template/blob/master/README.md), Please make sure you update your readme file to the [new readme file ](https://github.com/microverseinc/readme-template/blob/master/README.md)to get your project approved. :pray: